src/drivers/gps/gps.cpp

```diff
@@ -510,6 +510,8 @@ int GPS::setBaudrate(unsigned baud)
 	int speed;
 
 	switch (baud) {
+	case 4800:   speed = B4800;   break;
```

Comment:
@LorenzMeier I didn't know which one it is so I added both, I guess it does no harm?